README.md

```diff
@@ -62,14 +77,36 @@ There are _some_ sense to the default keybindings:
 Although initially designed for giving lectures, the whiteboard controls are pretty good.
 I might add support for extending infinitely in several directions, in the style of a regular whiteboard app.
 
+## Frequently Asked Questions
```

Comment:
Not Actually Frequently Asked. for style consistency headings should be in sentence case